### ISSUE TITLE
[BUG] Version info contains uninteded wordmarks

### ISSUE THREAD

**Issue by matmair** [inventree/InvenTree#3373]
**Describe the bug**
The `copy version info` button contains some unnecessary text

**Steps to Reproduce**

1. Copy version info with button
2. Paste somewhere and find copy instruction added at the end and a header at the beginning

**Expected behavior**
Clean text

**Deployment Method**
- [x] Docker
- [x] Bare Metal

**Version Information**
```
# Version Information:
InvenTree-Version: 0.8.0 dev
Django Version: 3.2.14
Commit Hash: afcd60b
Commit Date: 2022-07-20
Database: postgresql
Debug-Mode: False
Deployed using Docker: True

            
                 Versionsinformationen kopieren
```

You see what I did there? 😇 

**Ref:**
#3371

**Comment by miggland** [comment]
Just checked, and this does not happen on my InvenTree running 0.7.6. It does happen on the demo, development branch 0.8.0. See results:

```
# Version Information:
InvenTree-Version: 0.7.6
Django Version: 3.2.13


Database: postgresql
Debug-Mode: False
Deployed using Docker: True

```

```
# Version Information:
InvenTree-Version: 0.8.0 dev
Django Version: 3.2.14
Commit Hash: 164846fa7
Commit Date: 2022-07-24
Database: postgresql
Debug-Mode: True
Deployed using Docker: True
#

            
                 copy version information
```

**Comment by SchrodingersGat** [comment]
Fixed in #3425
